tests/features/sync.feature

```diff
@@ -15,22 +20,37 @@ Feature: Sync
   Scenario: Queue an entire category of posts for import
     Given I push all posts from the category sitemap "http://www.shortlist.com/sitemap/food-drink.xml" with the update method as "delete-insert" to the queue
     Then I should have a valid queue ID
-    Then I purge the queue
 
 
-  Scenario: Sync a post from url
+  Scenario: Sync a new post from url
     Given I process the queue action json '{"job":"importUrl","data":{"url":"http:\/\/www.shortlist.com\/food-drink\/michelin-star-restaurants-odd-unusual-world-uk-guide-food","onExistAction":"update"}}'
-    Then I should have imported the "michelin-star-restaurants-odd-unusual-world-uk-guide-food" post
+    Then I should have imported the "michelin-star-restaurants-odd-unusual-world-uk-guide-food-2" post
 
   Scenario: Push category import to queue
     Given I process the queue action json '{"job":"importCategory","data":{"url":"http:\/\/www.shortlist.com\/sitemap\/entertainment\/48-hours-to.xml","onExistAction":"update"}}'
     Then I should have an array of queue IDs
 
 
+  Scenario: Sync a post from url using update
+    Given I process the queue action json '{"job":"importUrl","data":{"url":"http:\/\/www.shortlist.com\/food-drink\/michelin-star-restaurants-odd-unusual-world-uk-guide-food","onExistAction":"update"}}'
+    Then I should have imported the "michelin-star-restaurants-odd-unusual-world-uk-guide-food-2" post
+    And I should have updated the post updated time
+
+  Scenario: Sync a post from url using delete-insert
+    Given I process the queue action json '{"job":"importUrl","data":{"url":"http:\/\/www.shortlist.com\/food-drink\/michelin-star-restaurants-odd-unusual-world-uk-guide-food","onExistAction":"delete-insert"}}'
+    Then I should have imported the "michelin-star-restaurants-odd-unusual-world-uk-guide-food-2" post
+    And I should have updated the post created time
+
+  Scenario: Sync a post from url using skip
+    Given I process the queue action json '{"job":"importUrl","data":{"url":"http:\/\/www.shortlist.com\/food-drink\/michelin-star-restaurants-odd-unusual-world-uk-guide-food","onExistAction":"skip"}}'
+    Then I should have imported the "michelin-star-restaurants-odd-unusual-world-uk-guide-food-2" post
+    And I should have the identical the post created and updated time
+
+
   Scenario: Get the full import status
     Given I retrieve the full import status
-    Then I should see a couple of imported out of at least 10000
+    Then I should see 100 imported out of at least 10000
```

Comment:
How is it `100`? I can't see the code that will consistently have the 100 imported posts in the DB?